upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1024,7 +1024,7 @@ func (c *ApplyClusterCmd) validateKopsVersion() error {
 		fmt.Println("")
 		fmt.Printf("More information: %s\n", buildPermalink("upgrade_kops", recommended.String()))
 		fmt.Println("")
-		fmt.Printf(starline)
+		fmt.Println(starline)
```

Comment:
Sorry - I think I just approved an earlier PR that is now conflicting here.  You might want to rebase / drop the changes in this file, as I think this file now passes staticcheck.

Reply:
no matter.
i will update it.
thanks